### PR TITLE
fixed : description

### PR DESCRIPTION
**Pull request by Sharmaryan**

### CODE REVIEW

File: styles/componentContainer/container.css [hunk 1]
```diff
@@ -16,11 +16,10 @@
 .inner-container-desc {
   padding: 1rem;
 }
-/* .inner-container-code{
-padding: 1rem;
-} */
+
 .code-snippet{
    width: 100%; 
    max-height: 300px;
```
Comment (zabihshaik) [inline styles/componentContainer/container.css]: better to use rem

File: styles/navigation/nav.html [hunk 2]
```diff
@@ -39,9 +39,7 @@ <h1>gotham<span>UI</span></h1>
       <div class="inner-container">
         <h3 class="inner-container-types">Navigation</h3>
         <p class="inner-container-desc">
-          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ratione
-          aliquam illum dicta sapiente possimus in inventore consequuntur
-          aperiam amet recusandae?
+         in order to navigate to different pages, navigaton bar can be used
```
Comment (zabihshaik) [inline styles/navigation/nav.html]: Typo of Navigation

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just a couple of comments.

File: styles/navigation/nav.html (hunk 2 above)
Comment (Sharmaryan) [reply styles/navigation/nav.html]: sure changing it

File: styles/componentContainer/container.css (hunk 1 above)
Comment (Sharmaryan) [reply styles/componentContainer/container.css]: sure

File: styles/card/card.css [hunk 3]
```diff
@@ -1,19 +1,11 @@
...
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
-}
-
 :root {
   --primary-color: white;
   --secondary-color: rgb(15, 1, 1);
   --default: rgb(53, 52, 52);
 }
```
Comment (MaruthiPothuganti) [inline styles/card/card.css:5]: This can be moved to a seperate a css file and that can be imported in any file

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested one change. Everything else looks good.

File: styles/card/card.css (hunk 3 above)
Comment (Sharmaryan) [reply styles/card/card.css:5]: sure will do that

**Review by chanchal-panpaliya — APPROVED** [review]
fixed navbar and links . code looks good .

**Comment by Sharmaryan** [comment]
> fixed navbar and links . code looks good .

fixed the links but will fix navbar later

**Review by Himanshu-Dhawale — APPROVED** [review]

**Pull request merged**